src/components/homepage-client.tsx

```
@@ -176,29 +134,10 @@ export function HomepageClient({
           currentSection={currentSection}
         />
 
-        <main id="main-content">
+        <main id="main-content" role="main">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<main>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)